### DISCUSSION TITLE
ZKSYNC DEPLOYMENT ERROR

### DISCUSSION THREAD

**Discussion by GeneralPharoah** [Cyfrin/foundry-full-course-cu#3124] [General]
Hey guys, I've been trying hard to deploy to zksync but I'm always met with an error whenever I try to compile my simplestorage.sol on zksync even after using the latest version. Please anyone who can help me out?

**Comment by H-code9** [comment]
Hi @GeneralPharoah,

1. navigate to the file explorer.
2. create a new folder named contracts.
3. copy your solidity file or your contract file and drop it in the contract folder.
4. compile your contract again, deploy and verify. Goodluck!!

**Comment by EngrPips** [comment]
Great heads-up

**Reply by H-code9** [reply]
Thanks @EngrPips Looking forward to learning from you.

**Reply by EngrPips** [reply]
I am convinced I will learn a lot from you too.

**Reply by H-code9** [reply]
I'm Humbled by your gesture, Looking forward to connecting with you, Best regards.
